ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -777,64 +723,56 @@ S3AddReference(WT_STORAGE_SOURCE *storageSource)
     return (0);
 }
 
-/*
- * S3Terminate --
- *     Discard any resources on termination.
- */
+// S3Terminate --
```

Comment:
Function header, I think we should remove it and just go straight to what the function does.

Reply:
Fixed.